src/app/searchBar.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { AutoComplete } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+import { Button } from "antd";
+
+const SearchBar: React.FC<{
+  search: string;
+  setSearch: any;
+  onClick: () => void;
```

Comment:
Do we need does functions?

What is a responsibility of this component?
Why is he told what happens of onClick and onKeyDown? Shouldn't it be his own decision?